src/ray/gcs/redis_context.cc

```diff
@@ -405,7 +405,7 @@ std::optional<std::pair<std::string, int>> ParseIffMovedError(
     return std::nullopt;
   }
   RAY_CHECK_EQ(parts.size(), 3u);
-  std::vector<std::string> ip_port = absl::StrSplit(parts[2], ":");
+  std::vector<std::string> ip_port = SplitIpPort(parts[2]);
```

Comment:
Looks like this still assumes IPv4 addresses or use split(':'). Without corresponding changes, services started in pure-IPv6 mode may fail to bind or mis-advertise their ports?